Makefile

```diff
@@ -0,0 +1,29 @@
+# HTML files that are generated from Markdown sources.
+HTML_FROM_MD_TARGETS=README.html README-with-toc.html
+
+.PHONY: all
+all: spec.html $(HTML_FROM_MD_TARGETS)
+
+.PHONY: clean
+clean:
+	-rm spec.html
+	-rm $(HTML_FROM_MD_TARGETS)
+
+# Updates Bikeshed's datafiles. Run this regularly to ensure you're not linking
+# out to stale specs. https://speced.github.io/bikeshed/#cli-update
+.PHONY: update
+update:
+	bikeshed update
+
+spec.html: spec.bs
+	bikeshed --die-on=everything spec $< $@
```

Comment:
Do we want to die on everything as the default?

Reply:
I like having a really strict local build and it hasn't caused issues so far. If it's annoying, we can always weaken this?

Reply:
Personally, I find it easier to have all the errors listed at once, but I'm also happy to just run `bikeshed`, so we can keep this.

Reply:
Oh, actually `--die-on` just makes the warnings fatal. Bikeshed will still emit all errors because  `--die-when` defaults to `late` ([docs](https://speced.github.io/bikeshed/#:~:text=If%20unspecified%2C%20this%20defaults%20to%20late.)).

Reply:
Empirically, before the last commit landed, I get one dfn error and three link errors when I call `bikeshed spec`, but no link errors when I call `bikeshed --die-on=everything spec`, but not certain why :)